### PR TITLE
Backpropagation  new entry

### LINKED ISSUE #2134: [Term Entry] AI Backpropagation
### Reason/inspiration (optional)

We would like a new term entry for the concept of Backpropagation.

The entry should go in a new file in docs/content/ai/concepts/neural-networks/terms/backpropagation/backpropagation.md

The entry should
- Describe what Backpropagation is.
- Explain how it is used in an Artificial Neural Network.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Himasnhu-AT**
Description
closes #2134 

Included a file in docs/content/ai/concepts/neural-networks/terms/linear-activation-function.md. the reason for it being that I think the linear function is a simple but very powerful tool that you can use in AI/ML to improve your model. As an aspiring AI Researcher, I thought, why not write it and improve my knowledge while contributing to this amazing community?

Type of Change
Adding a new entry
Checklist
- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked main branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes # in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Himasnhu-AT I did a first review and left several comments. Most important is your images were sourced from other websites. We do not permit plagiarism in docs and these images must be removed and/or replaced with original content.

Another thing, is that the Python example is more than we want in a typical Docs entry. Docs are supposed to be short and easily digestible, not full blown tutorials. I'd like to try this entry without it.

**Comment by Himasnhu-AT** [comment]
@SSwiniarski the changes are made. 
As for the images, I made them originally on my local machine while utilizing the references from the web.

**Comment by SSwiniarski** [comment]
> @SSwiniarski the changes are made. As for the images, I made them originally on my local machine while utilizing the references from the web.

@Himasnhu-AT Because the images are exact copies of what's on the web, they cannot be used. Please either remove them or come up with new ones.

**Comment by Himasnhu-AT** [comment]
@SSwiniarski i have updated the text, removing image references

**Review by SSwiniarski — APPROVED** [review]
@Himasnhu-AT Looks good for a second maintainer review.

**Review by KTom101 — APPROVED** [review]
@Himasnhu-AT LGTM 👍

**Comment by KTom101** [comment]
@Himasnhu-AT So you know, one of the checks failed because the file didn't have the markdown extension `.md`. I added the extension and ran `npm run format:all` to format the file so the checks pass.

**Comment by KTom101** [comment]
@Himasnhu-AT https://www.codecademy.com/resources/docs/ai/neural-networks/backpropagation

You're currently listed as anonymous:

Screenshot 2023-07-03 at 3 12 33 PM

If you want to remain anonymous, there is nothing more you need to do. If you wish to be listed as a contributor, please follow the instructions [in this forum discussion](https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036).

cc @yangc95
